### PR TITLE
Get fleets api

### PR DESCRIPTION
**Pull request by criSiles**

### CODE REVIEW

**Review by gracevivian04 — APPROVED** [review]
reviewed :)

**Review by LuciaSS92 — APPROVED** [review]
seems to be a repeated line at [models/Fleet.js]

**Comment by criSiles** [comment]
Could you tell me which one or even better put the comment in the code? I didn't find it.

File: models/Fleet.js [hunk 1]
```diff
@@ -10,7 +10,22 @@ const fleetSchema = mongoose.Schema ({
...
+    query.limit(limit);
+    query.select(fields);
+    query.sort(sort);
+    return query.exec();
+  };
+
 const Fleet = mongoose.model("Fleet", fleetSchema);
 
 // Export the User model
-module.exports = Fleet;
```
Comment (LuSo77) [inline models/Fleet.js:16]: @criSiles this little section in red

**Comment by gracevivian04** [comment]
> Could you tell me which one or even better put the comment in the code? I didn't find it.

eee sorry! I was supposed to click approved not comment - all good 👍

**Pull request merged**